### PR TITLE
Correcting incorrect closing double quotes in pagination Nunjucks

### PR DESCRIPTION
**Pull request by JoPintoPaul**
Spotted an incorrectly placed closing double quote in the template for the new Pagination component.

### CODE REVIEW

**Review by querkmachine — APPROVED** [review]
Looks good! Probably wanna stick a changelog entry onto this too before merging.

**Comment by JoPintoPaul** [comment]
> Looks good! Probably wanna stick a changelog entry onto this too before merging.

@querkmachine Would you like me to update the changelog on this PR, or will you folks do that?

**Comment by querkmachine** [comment]
@JoPintoPaul Either you can update the PR yourself or we can add the changelog entry, if you'd prefer.

**Comment by JoPintoPaul** [comment]
@querkmachine Updated PR with entry in CHANGELOG.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- [Correcting incorrect closing double quotes in pagination Nunjucks](https://github.com/alphagov/govuk-frontend/pull/3156)
```
Comment (owenatgov) [inline CHANGELOG.md:5]: Massive nitpick but we try to put fixes under the '## Fixes` heading below. Would you mind moving this to a new line under line 30? Happy to do this for you if you prefer.

**Pull request merged**